test/scripts/test_gas_diff_stats.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+import unittest
+from textwrap import dedent
+
+# NOTE: This test file file only works with scripts/ added to PYTHONPATH so pylint can't find the imports
+# pragma pylint: disable=import-error
+from gas_diff_stats import collect_statistics
+# pragma pylint: enable=import-error
+
+class TestGasDiffStats(unittest.TestCase):
```

Comment:
Isn't this missing a main?

```python
if __name__ == '__main__':
    unittest.main()
```
How we are supposed to run this tests? Shouldn't we add it to the CI or something?